todo_app/view_model.py

```diff
@@ -0,0 +1,38 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+        self.todo_items = items
+        self.doing_items = items
+        self.done_items = items
+
+    @property
+    def items(self):
+        return self._items
+
+    @items.setter
+    def items(self, items):
+        self._items = items
+
+    @property
+    def todo_items(self):
+        return self._todo_items
+    
+    @todo_items.setter
+    def todo_items(self, items):
+        self._todo_items = [item for item in items if item.status.lower() == 'to do']
```

Comment:
This is a novel way of doing it 🙂 

How about:
```suggestion
    @property
    def todo_items(self):
        return [item for item in self._items if item.status.lower() == 'to do']
```
and getting rid of the `self.todo_items = items` in the constructor. I think this is what Jack was getting at.

That's simpler and and means someone else can't accidentally assign to `todo_items`